src/site/content/en/blog/performance_tool_score_diffs.md

```diff
@@ -0,0 +1,179 @@
+- (as mentioned before) Page "load is not a single moment in time—it's an experience that no one metric can fully capture" [<a href="https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics">source</a>]
+- Similarly, no single metric in isolation is able to give a meaningful indication of the health of the web (critical for benchmarking).
+- Depending on which metric is chosen, a very different picture of page speed is presented (each requiring different optimizations to improve).
+
+## Performance Scores within Tools
+
+The performance score (top level gauge, 1-100 score) in PSI and Lighthouse is calculated from a weighted blend of multiple metrics to summarize a website's/web app's speed. This <a href="https://docs.google.com/spreadsheets/d/1sH_T4G_RZAg4CpcV1bT-tmUegBdBpCOOwsdzqtWnO4U/edit#gid=0">weighted blend</a> of metrics that together constitute the performance score in both PSI and LH (as of May 2019, <a href="https://github.com/GoogleChrome/lighthouse/releases">LH v5.0</a>) is as follows:
+- 3X - First Contentful Paint (FCP)
+- 1X - First Meaningful Paint (FMP)
+- 2X - First CPU Idle (FCPUI)
+- 5X - Time to Interactive (TTI)
+- 4X - Speed index (SI)
+- 0X - Max Potential First Input Delay (MPFID)
+
+<i>[Reasons for variance of scores between report runs of Lighthouse and PSI can be found <a href="https://docs.google.com/document/d/1AujmeKvBhzr-d8IsB7zPeS-vOtxCdw2GnspKpxJ7d_I/edit">here</a>.]</i>
+
+TMS uses only FCP (from both Lighthouse and from the CrUX) to provide both site and page speed scores.
+
+<div class="w-table-wrapper">
+  <table>
+    <thead>
+      <tr>
+        <th>Tool Name</th>
+        <th>Lab / Field Tool</th>
+        <th>Lab Metrics*</th>
+        <th>Field Metrics*</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><a href="https://www.thinkwithgoogle.com/intl/en-gb/feature/testmysite">Test My Site</a></td>
+        <td>Lab & Field</td>
+        <td>FCP**</td>
+        <td>FCP**</td>
+      </tr>
+      <tr>
+        <td><a href="https://developers.google.com/speed/pagespeed/insights/">PageSpeed Insights</a></td>
+        <td>Lab & Field</td>
+        <td>FCP, FMP, FCPUI, TTI, SI</td>
+        <td>FCP, FID</td>
+      </tr>
+      <tr>
+        <td><a href="https://developers.google.com/web/tools/lighthouse/">Lighthouse</a></td>
+        <td>Lab</td>
+        <td>FCP, FMP, FCPUI, TTI, SI</td>
+        <td>---</td>
+      </tr>
+    </tbody>
+    <caption>Table 1 — Performance metrics considered in score (by tool).</caption>
+  </table>
+  <p>*other metrics might be measured by the tools, but they aren’t included in performance/speed score calculations<br>**TMS uses FID as a tiebreaker in cases where FCP between compared sites is the same (but FCP is the lead metric)</p>
+</div>
+
+
+# Tool Sections: Breakdown
```

Comment:
Usually we only allow 1 h1 per page which is the article title.

I'd suggest switching this to an h2 using `##` and update the subsequent section headers to h3s